draft-wkumari-intarea-safe-limited-domains.md

```diff
@@ -109,116 +117,163 @@ Notably, in {{RFC8799}} Section 2, states:
+Extended EtherType, is that assignment of such an EtherType is significantly
+more difficult than assignment of an Extended EtherType IANA protocol number.
+As discussed in {{RFC9542}}, a specific EtherType can only be assigned by the
+IEEE Registration Authority under the following policy: "Since EtherTypes are a
+fairly scarce resource, the IEEE RAC has let us know that they will not assign
+a new EtherType to a new IETF protocol specification until the IESG has
+approved the protocol specification for publication as an RFC.  In exceptional
+cases, the IEEE RA is willing to consider "early allocation" of an EtherType
+for an IETF protocol that is still under development as long as the request
+comes from and has been vetted by the IESG." ({{RFC9542}} Appendix B.1, citing
+{{IESG_EtherType}})
+
+During development and testing, a protocol can use a "Local Experimental
+Ethertype" (0x88b5 and 0x88b6 - {{IANA_EtherType}}).  Once the protocol is
 approved for publication, the IESG can request an EtherType from the IEEE.
+However, there is always a risk of some implementation using a Local
+Experimental EtherType not getting updated causing conflicts with a later
+different use of that experimental EtherType.
 
-For discussion: or simply defining one single EtherType for this testing?
-I.e., IPv4 and IPv6 can be identified by their first 4 bits.
-
-[ Editor note: EtherTypes are a scarce resource, and so we need to be
-careful about how we use them. It is likely that there will only
-be a very limited (sorry!) number of protocols that need to be protected in
-this way (on the order of 3 or 4).
+The primary advantage of using a specific EtherType is the saving of 5 bytes
+relative to the use of the Extended EtherType with a protocol number under the
+IANA OUI.
 
-However, it is worth considering if there are other ways to achieve the same
-goal. An option would be to set aside a single EtherType, and then have a
-registry of "limited sub-EtherTypes" that are assigned by IANA. This would
-allow us to protect a large number of protocols, while only using a single
-EtherType, but would require something that looks more like a new L2 header. ]
-
-
-Another option to make a limited-domain protocol fail-closed is to use an
-identifier under the IANA OUI (00-00-5E) as explained in {{RFC9542}}.
-
-[Editor note: This is a good idea, but it is not clear if it is practical. This
-above would need a bunch more text / discussion. It ends up being a bit like
-the "limited sub-EtherTypes" idea above, but with the additional complexity of
-not having the MAC address be the "normal" MAC address of the device that you
-are sending the traffic to. This will require more discussion with the WG, and
-the IEEE liaisons. ]
 
 
 # Security Considerations
 
-TODO Security
+Protocols are designated as "limited domain" because something bad happens if
+they leak outside of a domain with unified management. For example, VLAN or VPN
```

Comment:
```suggestion
Protocols are designated as "limited domain" because something unexpected might happen if
they leak outside of a domain with unified management. For example, VLAN or VPN
```